packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -0,0 +1,262 @@
+<template>
+    <li
+        :class="['c-nav-list-item has-sublist', {
+            'open': countrySelectorIsOpen
+        }]"
+        v-on="isBelowMid ? null : { mouseover: openCountrySelector, mouseleave: closeCountrySelector }"
+        @keyup.esc="closeCountrySelector">
+        <a
```

Comment:
This one should be a button with type set to `button`. `<button type='button'>`

Reply:
Oh, yes, I just copied it from other nav tabs to start with and forgot to change :+1: